### ISSUE TITLE
Cannot start REPL

### ISSUE THREAD

**Issue by simax** [cursiveclojure/cursive#956]
Hi Colin,

You replied to a post I originally put on the IntelliJ bug list, but asked me in future to post issues here. In my original post on the IntelliJ bug list I somehow managed to mangle the error message So as clarification, this is the error message I get when I try to run a Clojure Repl in IntelliJ 14.1.4 on Windows 8.1:

Error:Cannot run program "\Library\Java\JavaVirtualMachines\jdk1.7.0_51.jdk\Contents\Home\bin\java" (in directory "C:\Users\simonlomax\.IdeaIC14\system\compile-server"): CreateProcess error=2, The system cannot find the file specified.
 
Any idea what's wrong?

**Comment by simax** [comment]
Worked out what happened now. I use IntelliJ on a MacBook Pro and on a Windows Laptop. I Exported setting from my MacBook Pro and imported them onto the Windows laptop. I must also have elected to copy the Project settings which is why I ended up with the paths from OSX in my windows IntelliJ settings. I fixed the problem by setting the JDK home path to 'C:\Program Files\Java\jdk1.7.0_79' in Project Structure settings.
